pkg/api/server/register_volumes.go

```diff
@@ -102,5 +103,35 @@ func (s *APIServer) registerVolumeHandlers(r *mux.Router) error {
 	//   500:
 	//     $ref: "#/responses/InternalError"
 	r.Handle(VersionedPath("/libpod/volumes/{name}"), s.APIHandler(libpod.RemoveVolume)).Methods(http.MethodDelete)
+
+	/*
+	 * Docker compatibility endpoints
+	 */
+
+	// swagger:operation GET /volumes volumes listVolumes
+	// ---
+	// summary: List volumes
+	// description: Returns a list of volume
+	// produces:
+	// - application/json
+	// parameters:
+	//  - in: query
+	//    name: filters
+	//    type: string
+	//    description: |
+	//      JSON encoded value of the filters (a map[string][]string) to process on the volumes list. Available filters:
+	//        - driver=<volume-driver-name> Matches volumes based on their driver.
+	//        - label=<key> or label=<key>:<value> Matches volumes based on the presence of a label alone or a label and a value.
+	//        - name=<volume-name> Matches all of volume name.
+	//
+	//      Note:
+	//        The boolean `dangling` filter is not yet implemented for this endpoint.
+	// responses:
+	//   '200':
+	//     "$ref": "#/responses/DockerVolumeList"
+	//   '500':
+	//     "$ref": "#/responses/InternalError"
+	r.Handle(VersionedPath("/volumes"), s.APIHandler(compat.ListVolumes)).Methods(http.MethodGet)
```

Comment:
From the network compat endpoint registration, it seems like I might also want unversioned path handlers here?

Reply:
There are unversioned path handlers now